src/stories/tests-factory/create-smoke-scenarios.ts

```diff
@@ -11,18 +12,20 @@ function checkIsCasesWithName<T>(cases: CasesWithName<T> | Cases<T>): cases is C
 
 export const createSmokeScenarios = <Props extends {}>(
     baseProps: Props,
-    propsCases: {
-        [K in Partial<keyof Props>]: CasesWithName<Props[K]> | Cases<Props[K]>;
-    },
+    propsCases: Partial<{
+        [K in keyof Props]: CasesWithName<Props[K]> | Cases<Props[K]>;
+    }>,
     options?: Options,
 ) => {
     const scenarioDetails: ScenarioDetails = {
         tag: ['@smoke', ...(options?.additionalTags || [])],
     };
 
+    const scenarioName: ScenarioName = `smoke scenario${options?.scenarioName ? ` ${options?.scenarioName}` : ''}`;
```

Comment:
For a more declarative name of the generated test name

before

after

Reply:
What if we shorten this to `smoke [default]` or `smoke [view: action]`, omit scenario and props words